src/firefly/java/edu/caltech/ipac/firefly/server/query/ptf/PtfProcimsFileRetrieve.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
+ */
+package edu.caltech.ipac.firefly.server.query.ptf;
+
+import edu.caltech.ipac.firefly.data.ServerRequest;
+import edu.caltech.ipac.firefly.server.query.SearchProcessorImpl;
+import edu.caltech.ipac.firefly.server.query.URLFileInfoProcessor;
+import edu.caltech.ipac.firefly.server.util.QueryUtil;
+import edu.caltech.ipac.util.StringUtils;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import static edu.caltech.ipac.firefly.server.query.ptf.PtfFileRetrieve.IBE_HOST;
+
+/**
+ * Created by IntelliJ IDEA. User: wmi Date: Feb 15, 2011 Time: 1:40:11 PM To change this template use File | Settings |
+ * File Templates.
+ */
+
+
+@SearchProcessorImpl(id = "PtfProcimsFileRetrieve")
```

Comment:
+@SearchProcessorImpl(id = "PtfProcimsFileRetrieve")
this processor id is the same with the one in ife but the getBaseURL parameter is different, in ife(gwt) the host, schemaGroup, schema and table are read in from the xml configuration. In firefly the params are set in the PTFPlotRequests and ptfDownloadOptPanel, "mission" is equivalent to schemaGroup. would it be confusion, when we build TST under irsaviewer? the two processor (one in firefly, the other in ife) with the same processor id. I would suggest using PtfTSProcimsFileRetrieve